src/nix/store.cc

```diff
@@ -8,6 +8,7 @@ struct CmdStore : NixMultiCommand
     {
         aliases = {
             {"ping", { AliasStatus::Deprecated, {"info"}}},
+            {"optimize", { AliasStatus::AcceptedShorthand, {"optimise"}}},
```

Comment:
nitpick(non-blocking): Cool! I was also thinking, maybe we could have another `AliasStatus` for stuff like this, since it's not really a shorthand.

```suggestion
            {"optimize", { AliasStatus::LinguisticVariation, {"optimise"}}},
```

And then in `src/libutil/include/nix/util/args.hh`:

```diff
diff --git c/src/libutil/include/nix/util/args.hh i/src/libutil/include/nix/util/args.hh
index 463270374..9caa70a6d 100644
--- c/src/libutil/include/nix/util/args.hh
+++ i/src/libutil/include/nix/util/args.hh
@@ -398,6 +398,8 @@ public:
         AcceptedShorthand,
         /** Aliases that will go away */
         Deprecated,
+        /** Aliases for linguistic variations */
+        LinguisticVariation,
     };

     /** An alias, except for the original syntax, which is in the map key. */
```